aws-crt-kotlin/native/src/aws/sdk/kotlin/crt/CRTNative.kt

```diff
@@ -2,30 +2,53 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
+
 package aws.sdk.kotlin.crt
 
+import kotlinx.atomicfu.atomic
+import kotlinx.cinterop.ExperimentalForeignApi
+import kotlinx.cinterop.convert
+import kotlinx.cinterop.staticCFunction
+import kotlinx.cinterop.toKString
+import libcrt.*
+import platform.posix.atexit
+
+@OptIn(ExperimentalForeignApi::class)
 public actual object CRT {
```

Comment:
**Suggestion**: Some new tests that test basic sanity would be good (e.g. `Crt.errorString(...)`, `initRuntime`, etc